lib/cf/broker/src/test/plans/plan-builder-test.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+const { omit } = require('underscore');
+const createPlanBuilder = require('../../plans/plan-builder');
+
+const isFunction = (value) => typeof value === 'function';
+
+describe('PlanBuilder', () => {
+  const planId = 'plan-id';
+
+  let planBuilder;
+
+  beforeEach(() => {
+    planBuilder = createPlanBuilder();
+  });
+
+  describe('createMeteringPlan', () => {
+    const meteringPlan = {
+      plan_id: 'user-provided-plan',
+      data: 1
+    };
+
+    context('when no plan is provided', () => {
+      it('should return default plan', () => {
+        const plan = planBuilder.createMeteringPlan(planId);
+
+        expect(plan.plan_id).to.equal(planId);
+        expect(plan.measures).to.deep.equal([{
+          name: 'sampleName',
+          unit: 'sampleUnit'
+        }]);
+
+        expect(plan.metrics.length).to.equal(1);
+        expect(omit(plan.metrics[0], 'meter', 'accumulate', 'aggregate', 'summarize')).to.deep.equal({
```

Comment:
Can we use `expect(plan.metrics[0]).to.deep.include.members({ ... })` here? Do we need to check for additional fields not present in members?